src/control/Control.Layers.js

```diff
@@ -346,7 +350,18 @@ L.Control.Layers = L.Control.extend({
 			                 (layer.options.maxZoom !== undefined && zoom > layer.options.maxZoom);
 
 		}
+	},
+
+	_expand: function () {
+		// Backward compatibility, remove me in 1.1.

```

Comment:
Don't know if we've deprecated something before, and how we did it, but perhaps put a `console.warn` here and in `_collapse`, telling we're removing this in 1.1 and that you should use `collapse`/`expand`?